Mathlib/Algebra/Polynomial/Derivative.lean

```diff
@@ -146,6 +146,9 @@ theorem iterate_derivative_C_mul (a : R) (p : R[X]) (k : ℕ) :
     derivative^[k] (C a * p) = C a * derivative^[k] p := by
   simp_rw [← smul_eq_C_mul, iterate_derivative_smul]
 
+theorem derivative_C_mul (a : R) (p : R[X]) :
```

Comment:
```suggestion
@[simp]
theorem derivative_C_mul (a : R) (p : R[X]) :
```

Reply:
I found that this gives the following build(lint?) error [link](https://github.com/leanprover-community/mathlib4/actions/runs/11844166189/job/33006753558?pr=19048)
```
/- The `simpNF` linter reports:
SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%[20](https://github.com/leanprover-community/mathlib4/actions/runs/11844166189/job/33006753558?pr=19048#step:24:21)form -/
-- Mathlib.Algebra.Polynomial.Derivative
Error: ././././Mathlib/Algebra/Polynomial/Derivative.lean:149:1: error: @Polynomial.derivative_C_mul simp can prove this:
  by simp only [@Polynomial.derivative_mul, @Polynomial.derivative_C, @zero_mul, @zero_add]
One of the lemmas above could be a duplicate.
If that's not the case try reordering lemmas or adding @[priority].
```
How can I fix this?

Reply:
Ah then ignore Johan's suggestion

Reply:
The error message is basically telling you that "this thing you're marking as `@[simp]` can itself be solved with `simp` (move it down to line 300 and see), so either don't mark this as simp or you're doing something wrong!"